app/controllers/concerns/foreman/controller/parameters/compute_resource.rb

```diff
@@ -47,9 +47,6 @@ def compute_resource_params_filter
           :public_key,
           :uuid
 
-        # rackspace
-        filter.permit :region
```

Comment:
https://github.com/theforeman/foreman/blob/c86ed9c68adedaaff5562ddaa6e7229ec416427d/app/models/compute_resources/foreman/model/ec2.rb#L10

Doesn't AWS also use this?

Reply:
it's still permitted, on line 20